includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -0,0 +1,218 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Helper_Updater {
```

Comment:
Just a nitpick; some classes and methods throughout the PR are missing docblocks.